packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -336,6 +337,8 @@ const TldrawEditorWithLoadingStore = memo(function TldrawEditorBeforeLoading({
 	return <TldrawEditorWithReadyStore {...rest} store={store.store} user={user} />
 })
 
+const noAutoFocus = () => document.location.search.includes('preserveFocus')
```

Comment:
i'm worried that a library consumer might include this for some other reason and then break things. can we call it something horrible like `tldraw_preserveFocus_Sq06lHBj`?